src/deployment.jl

```diff
@@ -71,13 +89,19 @@ function deployment_area(max_n_corals::Int64, target_areas::Vector{Float64})::Tu
     end
 
     if density == 0.0
-        throw(DomainError("Could not determine adequate deployment density: $((RME_BASE_GRID_SIZE[] * req_area / sum(target_areas)))"))
+        throw(
+            DomainError(
+                "Could not determine adequate deployment density: $((RME_BASE_GRID_SIZE[] *sum(req_area) / sum(target_areas)))"
+            )
+        )
     end
 
     @info "Determined min. deployment density to be: $(density) / m²"
-
     deployment_area_pct = min((req_area / sum(target_areas)) * 100.0, 100.0)
 
+    # In RME versions higher than 1.0.28 density needs to be a vector with each element representing density per species
+    version_vec = version_check()
+    density = (version_vec[3] <= 28) && (version_vec[1] == 1) ? density : fill(density, 6)
```

Comment:
Ignoring the hardcoded indexing here, think about what would happen if the RME version advances to v1.2.28